### PR TITLE
Update unblockit proxies domain to *.one

### PR DESCRIPTION
**Pull request by ilike2burnthing**
I can't test Demonoid due to ongoing issues with Cloudflare and VPN/proxy usage (and separately, an ISP block), so if someone can test that for me please

Pirateiro has it's own issue, likely causing the unblockit failure as well - https://github.com/Jackett/Jackett/issues/7973

Re-re-fix of katcr.co's unblockit proxy (basically I was right the first time, seems to have had a temporary issue yesterday so I thought I'd made a mistake)

Everything else tested and working

### CODE REVIEW

**Comment by ilike2burnthing** [comment]
Ah completely forgot, sorry, I'll get on that.

**Comment by ilike2burnthing** [comment]
Huh, it failed when I was testing a couple of hours ago. It's back up now so I added it in.

**Comment by ilike2burnthing** [comment]
Spoke too soon. Keyword search is fine, empty search fails.

Links at the top to categories and 'recent' redirect in browser.

File: src/Jackett.Common/Definitions/torrentdownload.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   followredirect: true
   links:
     - https://www.torrentdownload.info/
-    - https://torrentdownload.unblockit.pro/
+    - https://torrentdownload.unblockit.one/
     - https://torrentdownload.black-mirror.xyz/
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/torrentdownload.yml:12]: you've missed legacylink for this one ;-)

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/torrentdownload.yml:12]: There's always one 😋

**Comment by ilike2burnthing** [comment]
I remembered how to test Demonoid et al for Jackett in Windows sandbox. Turns out it fails 😐

Do you want (me) to pull it in a new PR?

**Comment by garfield69** [comment]
yes, you're going to have to raise a new PR, since this PR has been merged now, and won't accept further updates.

**Comment by ilike2burnthing** [comment]
Nevermind, past me played it safe by putting it in legacylinks, I think because the proxy site had issues

**Pull request merged**